tests/packaging/pip_install_girder.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
This breaks some behavior in the script. If you look below, we explicitly test return values and print human readable error messages.